sql/src/test/java/org/opensearch/sql/sql/SQLServiceTest.java

```diff
@@ -83,6 +84,115 @@ public void onFailure(Exception e) {
         });
   }
 
+  @Test
+  public void canExecuteJsonFormatRequest() {
+    doAnswer(invocation -> {
+      ResponseListener<QueryResponse> listener = invocation.getArgument(1);
+      listener.onResponse(new QueryResponse(schema, Collections.emptyList(), ""));
+      return null;
+    }).when(queryService).execute(any(), any());
+
+    sqlService.execute(
+        new SQLQueryRequest(new JSONObject(), "SELECT FIELD FROM TABLE", QUERY, "json"),
+        new ResponseListener<QueryResponse>() {
+          @Override
+          public void onResponse(QueryResponse response) {
+            assertNotNull(response);
+          }
+
+          @Override
+          public void onFailure(Exception e) {
+            fail(e);
+          }
+        });
+  }
+
+  @Test
+  public void canThrowUnsupportedExceptionForAggregationJsonQuery() {
+    sqlService.execute(
+        new SQLQueryRequest(new JSONObject(),
+            "SELECT FIELD FROM TABLE GROUP BY FIELD", QUERY, "json"),
+        new ResponseListener<QueryResponse>() {
+          @Override
+          public void onResponse(QueryResponse response) {
+            assertNotNull(response);
+          }
+
+          @Override
+          public void onFailure(Exception e) {
+            assertTrue(e instanceof UnsupportedOperationException);
+          }
+        });
+  }
+
+  @Test
+  public void canThrowUnsupportedExceptionForAliasJsonQuery() {
+    sqlService.execute(
+        new SQLQueryRequest(new JSONObject(), "SELECT FIELD as FIELD FROM TABLE", QUERY, "json"),
+        new ResponseListener<QueryResponse>() {
+          @Override
+          public void onResponse(QueryResponse response) {
+            assertNotNull(response);
+          }
+
+          @Override
+          public void onFailure(Exception e) {
+            assertTrue(e instanceof UnsupportedOperationException);
+          }
+        });
+  }
+
+  @Test
+  public void canThrowUnsupportedExceptionForFunctionJsonQuery() {
+    sqlService.execute(
```

Comment:
This block of code is common to all the tests, isn't it? They only differ in query sent.

Can you put it in a utility method that's call by the tests?

You can make it even more compact by creating a parameterized test that takes a list of sql statement to test.

Reply:
Thanks, changed in e0e2365ec55512d6f41996c17e95ff773ed59515. Please let me know if you had something else in mind.